### DISCUSSION TITLE
There are various anomalies when using KernrlSU on pixel 7 pro

### DISCUSSION THREAD

**Discussion by JerryLYU233** [tiann/KernelSU#1943] [Q&A]
**简体中文**
在使用KernelSU修补Pixel 7 Pro的init_boot(非boot因为pixel只支持修补init_boot)时会出现各种奇怪问题，一下为症状：
1.安装一个模块后选择不重启直接退出安装另一个会报错
2.手机开机使用一段时间再安装模块也会报错
3.同样的，手机开机使用一段时间禁用亦或卸载模块会无法禁用/卸载
附上日志及图片:
**英文**
When using KernelSU to patch the init_boot (not boot because pixel only supports patching init_boot) of Pixel 7 Pro, various strange problems will occur. The following are the symptoms:
1. After installing a module, choose not to restart and directly exit to install another one, an error will be reported
2. The phone will be turned on and used for a while before installing a module, and an error will also be reported
3. Similarly, the phone will be turned on and used for a while, and the module will not be disabled/uninstalled
Attached log and pictures
[KernelSU_install_log_2024-08-05-17-06-16.log](https://github.com/user-attachments/files/16493536/KernelSU_install_log_2024-08-05-17-06-16.log)
![Screenshot_20240805-170107](https://github.com/user-attachments/assets/17d40507-a1f8-4b72-b272-1b9a03126a05)
![Screenshot_20240805-170112](https://github.com/user-attachments/assets/ac2efa9e-e60b-4f21-a70a-dc3034caa061)
![Screenshot_20240805-170126](https://github.com/user-attachments/assets/a3aeac3b-08e2-4290-bb78-53c1f7f968a2)
![Screenshot_20240805-170349](https://github.com/user-attachments/assets/fd5f7283-e7a3-46b3-9be9-e0c60d229c00)
[KernelSU_bugreport_2024-08-05_17_01.tar.gz](https://github.com/user-attachments/files/16493540/KernelSU_bugreport_2024-08-05_17_01.tar.gz)

**Comment by tiann** [comment]
把你那堆模块关了再试

**Reply by JerryLYU233** [reply]
那一大堆全关了再开机过一会儿也没用，打不开，要一开机赶紧操作，挺考验手速
